app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -310,13 +310,13 @@ public void setNotificationCount() {
      * Sets the visibility of the upload icon based on the number of failed and pending
      * contributions.
      */
-    public void setUploadIconVisibility() {
-        contributionController.getFailedAndPendingContributions();
-        contributionController.failedAndPendingContributionList.observe(getViewLifecycleOwner(),
-            list -> {
-                updateUploadIcon(list.size());
-            });
-    }
+//    public void setUploadIconVisibility() {
+//        contributionController.getFailedAndPendingContributions();
+//        contributionController.failedAndPendingContributionList.observe(getViewLifecycleOwner(),
```

Comment:
I think we could also comment the `failedAndPendingContributionList` member and the corresponding `getFailedAndPendingContributions` as I suppose they are not used for any other purpose.